_targets.R

```diff
@@ -1,8 +1,13 @@
 suppressPackageStartupMessages(library(dplyr))
+library(leaflet)
+library(leafpop)
+library(htmlwidgets)
 library(targets)
 library(tarchetypes)
 library(tibble)
 
+library(tidyr)
+
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval", "urbnmapr",
                             "rnaturalearth", "cowplot", "lubridate"))
```

Comment:
Just one note that there is some slight differences in behavior between library calls vs putting packages within the tar_option_set (e.g. they are then only loaded if any targets need to be built). I think generally putting packages that are just used in the targets makes sense, but in general either works.

Reply:
I totally forgot about there being two locations to load packages. I'll fix this before merging.